.appveyor.yml

```diff
@@ -0,0 +1,20 @@
+
+  matrix:
```

Comment:
`Error parsing appveyor.yml: (Line: 6, Col: 1, Idx: 64) - (Line: 6, Col: 1, Idx: 64): Did not find expected <document start>.`